lib/src/frame.dart

```diff
@@ -30,6 +30,12 @@ final _v8UrlLocation = new RegExp(r'^(.*):(\d+):(\d+)|native$');
 final _v8EvalLocation =
     new RegExp(r'^eval at (?:\S.*?) \((.*)\)(?:, .*?:\d+:\d+)?$');
 
+// foo._bar@http://pub.dartlang.org/stuff.js:18056:12
+// anonymous/<@http://pub.dartlang.org/stuff.js line 693 > function:3:40
```

Comment:
nit: replace function with Function (capital F)

also, consider adding here too the example with `eval` as well?

Reply:
replaced/added